.github/workflows/continuous-integration-workflow.yml

```
@@ -0,0 +1,57 @@
+name: CI
```

```suggestion
name: github-Linux
```
for consistency with "github-OSX" or similar because the checks currently read "CI / CI"